sdk/core/core-rest-pipeline/test/tracingPolicy.spec.ts

```diff
@@ -307,6 +313,7 @@ describe("tracingPolicy", function () {
     assert.lengthOf(mockTracer.getStartedSpans(), 1);
     const span = mockTracer.getStartedSpans()[0];
     assert.isTrue(span.didEnd());
+    assert.equal(span.getName(), "HTTPS PUT");
```

Comment:
should we also validate the `http.method` and `http.url` attributes while we're at it?

Reply:
Makes sense, I added a separate test to validate most of the attributes